package.json

```diff
@@ -20,11 +20,13 @@
     "e2e:playwright": "(set TS_NODE_PROJECT=./e2e-playwright/tsconfig.json) && playwright test --config=./e2e-playwright/playwright.config.ts",
     "e2e:playwright:ci": "(set TS_NODE_PROJECT=./e2e-playwright/tsconfig.json) && playwright test --config=./e2e-playwright/playwright.config.ci.ts",
     "e2e:playwright:ui": "(set TS_NODE_PROJECT=./e2e-playwright/tsconfig.json) && playwright test --ui --config=./e2e-playwright/playwright.config.ts",
+    "e2e:playwright:ci-browserstack": "(set TS_NODE_PROJECT=./e2e-playwright/tsconfig.json) && browserstack-node-sdk playwright test --config=./e2e-playwright/playwright.config.ci.ts",
     "e2e:wdio": "(set TS_NODE_PROJECT=./e2e-wdio/tsconfig.json) && wdio run ./e2e-wdio/wdio.conf.ts",
     "e2e:wdio:ci": "start-server-and-test start http://localhost:4200 e2e:wdio",
     "galen": "galen",
     "layout-tests:browserstack": "bash ./layout-tests/run-tests-on-browserstack.sh",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "npm update browserstack-node-sdk"
```

Comment:
Why is this postinstall needed as the browserstack-node-sdk is already part of the dependencies below?

Reply:
Must have been added when we installed the browserstack-node-sdk. We don't see the need to keep it and removed the line